404.html

```diff
@@ -1,4 +1,5 @@
 ---
```

Comment:
Not related to this file per se but favicon is missing, can we use logo for this?

Reply:
Done.